cabal.project

```diff
@@ -20,8 +20,9 @@ tests: True
 -- necessary to use a haskeline <0.8 with GHCs that ship with base >= 4.14
 allow-newer: haskeline:base
 
--- https://github.com/TomMD/entropy/issues/75
-constraints: entropy < 0.4.1.9
+-- develop with -Werror on, for consistency with internal Meta CI.
+program-options
+    ghc-options: -Werror -Wwarn=dodgy-imports
```

Comment:
Internally, dodgy-imports is an error not a warning.

Reply:
Yeah ok, unfortunately I'll have to use some `#ifdef`s because the Prelude exports changed between GHC versions. Really `-Wwarn=dodgy-imports` is the right thing because it avoids the need for CPP, but it's not a big deal.